ophyd/controls/area_detector.py

```diff
@@ -638,7 +638,6 @@ def configure(self, state=None):
         self._write_plugin('AutoSave', 1, self._file_plugin)
         self._write_plugin('NumCapture', 0, self._file_plugin)
         self._write_plugin('FileWriteMode', 2, self._file_plugin)
-        self._write_plugin('EnableCallbacks', 1, self._file_plugin)
```

Comment:
Not with this exact code path because things changed under it.

On Sat, Oct 31, 2015, 12:02 dchabot <notifications@github.com> wrote:

> In ophyd/controls/area_detector.py
> <https://github.com/NSLS-II/ophyd/pull/163#discussion_r43573194>:
>
> > @@ -638,7 +638,6 @@ def configure(self, state=None):
> >          self._write_plugin('AutoSave', 1, self._file_plugin)
> >          self._write_plugin('NumCapture', 0, self._file_plugin)
> >          self._write_plugin('FileWriteMode', 2, self._file_plugin)
> > -        self._write_plugin('EnableCallbacks', 1, self._file_plugin)
>
> Is this actually being used at 28id? Successfully?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NSLS-II/ophyd/pull/163/files#r43573194>.
>